kubernetes.yaml

```diff
@@ -80,6 +85,15 @@ data:
 
     repository   = scrapyd_k8s.repository.Remote
     launcher     = scrapyd_k8s.launcher.K8s
+    
+    num_lines_to_check = 2
+    
+    storage_provider = s3
+    
+    aws_region            = <aws_region>
+    aws_access_key_id     = <aws_access_key_id>
+    aws_secret_access_key = <aws_secret_access_key>
```

Comment:
It's bad practice to have secrets in a configmap. For kubernetes, we could mention a secret name (but then the daemon needs to have access to the secret), or perhaps better using environment variables, and then the manifest can set these from a secret (giving the flexibility to the deployment configuration). For Docker, we need to use the environment anyway.